doc/classes/RayCast2D.xml

```diff
@@ -44,7 +44,7 @@
 			</return>
 			<description>
 				Updates the collision information for the ray. Use this method to update the collision information immediately instead of waiting for the next [code]_physics_process[/code] call, for example if the ray or its parent has changed state.
-				[b]Note:[/b] [code]enabled == true[/code] is not required for this to work.
+				[b]Note:[/b] [code]enabled[/code] is not required for this to work.
```

Comment:
The previous sentence was not the clearest but now it's confusing as `[code]enabled[/code]` should refer to an argument of this method, which doesn't exist. Instead it's `[member enabled]` which we're referring to, so it should use the `[member]` hyperlink (but just mentioning the `enabled` property isn't a clear indication of whether it's actual ON or OFF, so this should probably be rephrased).

Reply:
I tweaked the documentation, I think it makes more sense now.